writeToStream.js

```diff
@@ -41,6 +41,7 @@ function generate (packet, stream) {
       return false
   }
 }
+generate.cacheNumbers = true
```

Comment:
I think we can use a setter here, and if it is setted to false, clear `numCache`.